src/lightning_app/cli/lightning_cli.py

```diff
@@ -380,6 +381,27 @@ def run_app(
 
     run.add_command(_run_model)
 
+
+@_main.command("open")
+@click.argument("path", type=str, default=".")
+@click.option(
+    "--cluster-id",
+    type=str,
+    default=None,
+    help="Open on a specific Lightning AI BYOC compute cluster",
+)
+@click.option("--name", help="The name to use for the CloudSpace", default="", type=str)
+def open(path: str, cluster_id: str, name: str) -> None:
+    """Open files or folders on the cloud."""
```

Comment:
Can we expand on this docstring? it's not clear what this does. maybe something like: "Open file or folders on your local machine in a lightning cloudspace?"